Gemfile

```diff
@@ -2,11 +2,11 @@ source 'https://rubygems.org'
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
 # Use rails for the application framework
-gem 'rails', '~> 5.2'
+gem 'rails', '~> 6.0'
 # Use Puma as the app server
-gem 'puma', '~> 3.11'
+gem 'puma', '~> 4.1'
 # Use SCSS for stylesheets
-gem 'sass-rails', '~> 5.0'
+gem 'sass-rails', '>= 6'
```

Comment:
Do we still have a thing where we couldn't deploy this for some reason?

Reply:
I don't think so, and I just pushed up it to be sure and it went fine 🤷‍♂️

Reply:
nice.. we have this pinned everywhere so seems like we can start unpinning.